### ISSUE TITLE
About composer.json

### ISSUE THREAD

**Issue by phoenixg** [michelf/php-markdown#138]
I have already have a composer.json file which content is : 
    {
        "require": {
            "torophp/torophp": "dev-master",
            "mustache/mustache": "dev-master"
        }
    }

How to include your composer.json content to mine?

**Comment by paazmaya** [comment]
Add the line `"michelf/php-markdown": "1.4.1"` in the `require` object of your `composer.json` file.
https://packagist.org/packages/michelf/php-markdown

@michelf perhaps this could be mentioned in the `README.md` ?

**Comment by GrahamCampbell** [comment]
You should not require a specific version like that. Use a **version constraint**.

**Comment by michelf** [comment]
Indeed, it's probably better to use a version constrain in most cases. I'm not too sure what to recommend though, beside that people should read the section about version numbering in the Readme.
